baseline_testing/testcase_bestest_air_scenario.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+"""
+This script demonstrates baseline testing of test case called "bestest_air".
+
+"""
+
+# GENERAL PACKAGE IMPORT
+# ----------------------
+import sys
+import os
+sys.path.insert(0, '/'.join(os.path.dirname(os.path.abspath(__file__))))
+from examples.python.interface import control_test
+import pandas as pd
+import json
+import itertools
+import datetime as dt
+import time
+
+
+def run(modelname, scenario, start_time = 0, warmup_period = 0, length = 48*3600,  plot=False):
+    """Run test case.
+    Parameters
+    ----------
+    modelname: str
+        Name of the testcase.
+        
+    scenario: dict, optional
+        Dictionary defining the predefined test scenario.
+        {'time_period': str, 'electricity_price': str}.
+        If specified, start_time, warmup_period, and length not used.
+        
+    start_time: int, optional
+        Simulation start time in seconds from midnight January 1st.
+        Not used if scenario defined.
+        
+    warmup_period: int, optional
+        Simulation warmup-period in seconds before start_time.
+        Not used if scenario defined.
+        
+    length: int, optional
+        Simulation duration in seconds (e.g., 24*3600 is a 1 day simulation).
+        Not used if scenario defined.
+
+    plot : bool, optional
+        True to plot timeseries results.
+        Default is False.
+
+    Returns
+    -------
+    kpi : dict
+        Dictionary of core KPI names and values.
+        {kpi_name : value}
+    res : dict
+        Dictionary of trajectories of inputs and outputs.
+    custom_kpi_result: dict
+        Dictionary of tracked custom KPI calculations.
+        Empty if no customized KPI calculations defined.
+
+    """
+
+    # RUN THE CONTROL TEST
+    # --------------------
+    control_module = 'examples.python.controllers.{}'.format(modelname)
+    step = 300
+    
+    # RUN THE CONTROL TEST
+    # --------------------
+    #start_time, warmup_period, length will not be used if scenario is specified.
+    kpi, df_res, custom_kpi_result, forecasts = control_test(control_module,start_time,warmup_period,length,
+                                                                  scenario=scenario,step=step)
+
+    # POST-PROCESS RESULTS
```

Comment:
I suggest removing the post-processing section as it is very test case specific and I don't see it as particularly useful in this baselining exercise.

Reply:
It has been removed.